### ISSUE TITLE
Shoulder apprehension test

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10997]
**Preferred term label:**
Shoulder apprehension test

**Synonyms**

**Definition (free text, please give PubMed ID)**
Visible apprehension noted when patient is laying supine with shoulder abducted to 90 degrees, elbow flexed to 90 degrees, and an external rotation pressure with combined downward pressure is applied on the upper arm

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Shoulder dislocation. Shoulder labrum tear

**Your nano-attribution (ORCID)**
https://www.physio-pedia.com/Apprehension_Test

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001228
- label: Shoulder apprehension test
- parents: [Abnormal shoulder physiology (HP:0033129)](https://hpo.jax.org/app/browse/term/HP:0033129)
- pmid: PMID:14977651
- def: The individual being investigated displays noticeable apprehension (unease) during a test for shoulder instability in which the individual is laying supine with shoulder abducted to 90 degrees, elbow flexed to 90 degrees, and an external rotation pressure with combined downward pressure is applied on the upper arm.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10997
- synonyms:
